website/docs/contributing/writing-guide.md

```diff
@@ -46,10 +46,21 @@ Use sentence case for headers: `### Some header with URL`, not `### Some Header
 Please use markdown file paths for links, not URL paths,
 because it works for both editors/IDEs (Ctrl/⌘+click works) and Docusaurus.
 Always add `.md` extension to the file paths.
+Use relative paths for links to files in the same directory or in a parent directory.
+
 Examples:
 
-- [file in the same directory](writing-guide.md)
-- [file in a parent directory](../telemetry.md)
+To link to file in a subdirectory, prefix with `./` to go down one directory level.
+
+- `[file in a subdirectory](./writing-guide.md)`
+
+To link to a file in the same directory, use the file name.
+
+- `[file in the same directory](writing-guide.md)`
```

Comment:
Something is not right there. `./writing-guide.md` and `writing-guide.md` are the same thing

Reply:
Yes, not the same thing but it's probably confusing. Wanted to demonstrate how to link to a sub-directory since there's none in that section. - `[file in a subdirectory](./writing-guide.md)`. I'll just update with something else

Reply:
But `./writing-guide.md` is not a subdirectory – that's the same directory. You probably wanted `./subdir/file.md` – but then why it can't be `subdir/file.md`

Reply:
That's correct! 👍🏽